plugins/spark/v3.5/src/test/java/org/apache/polaris/spark/SparkCatalogTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.spark;
+
+import static org.apache.iceberg.CatalogProperties.CATALOG_IMPL;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.Map;
+import java.util.UUID;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
+import org.apache.spark.sql.connector.catalog.Identifier;
+import org.apache.spark.sql.types.StructType;
+import org.apache.spark.sql.util.CaseInsensitiveStringMap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class SparkCatalogTest {
+  private SparkCatalog catalog;
+  private String catalogName;
+
+  @BeforeEach
+  public void setup() {
+    catalogName = "test_" + UUID.randomUUID();
+    Map<String, String> catalogConfig = Maps.newHashMap();
+    catalogConfig.put(CATALOG_IMPL, "org.apache.iceberg.inmemory.InMemoryCatalog");
+    catalogConfig.put("cache-enabled", "false");
+    catalog = new SparkCatalog();
+    catalog.initialize(catalogName, new CaseInsensitiveStringMap(catalogConfig));
+  }
+
+  @Test
+  public void testUnsupportedOperations() {
```

Comment:
I'm still not sure there is any benefit to having this test. If we just want a no-op test to prove everything compiled I would just try to init and not throw an error.